Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/computers_shipyard.yml

```
@@ -221,3 +221,27 @@
       state: shipyard_blackmarket
     - map: ["computerLayerKeys"]
       state: blackmarket_key
+
+- type: entity
+  id: ComputerShipyardSr
+  parent: [BaseStructureDisableToolUse, BaseStructureIndestructible, ComputerShipyard]
+  name: sr shipyard console
+  description: Used to buy vessels exclusive to the station representative
```

Capitalize proper nouns.
```suggestion
  name: SR shipyard console
  description: Used to buy vessels exclusive to the Station Representative.
```

---

I know how to grammar! This is following the existing convention for the other consoles.

---

Sounds like a good time to add periods then.  "nfsd" is its own issue - acronyms (ERT, CBURN, R&D...) are generally capitalized, and I think this should be consistent.

---

If only we could get any sort of consistency on nanotrasen vs. Nanotrasen vs. NanoTrasen.

---

Yeah, it's really frustrating, honestly. There are HUGE amounts of inconsistency when it comes to capitalisations and grammar in certain things, and I've been told to do things certain ways by maintainers on other forks. I've ended just following the majority to avoid getting bapped. I would MUCH rather have this properly capitalised and with the correct grammar, but I also want to follow the established convention.